.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     -   id: check-pylint-normal
         name: Check pylint (except unit tests)
         types: [file, python]
-        exclude: ^tests/garage/.*$  # exclude unit tests
+        exclude: ^tests/garage/.|tests/fixtures/.*$  # exclude unit tests
```

Comment:
actually my intention here was that test fixtures should also be documented. please remove this part.